### PR TITLE
Introduction to Deep Learing - Draft 

### PR DESCRIPTION
**Pull request by khanteymoori**
Dear @anuprulez 
It is a draft of introduction to deep learning tutorial, 
Simple dataset is used, and it is not complete.
Please modify it with the new data set, and I will try to add more about background and the theory behind.

### CODE REVIEW

File: topics/statistics/tutorials/classification-machinelearning/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+layout: tutorial_hands_on
+
+title: 'Classification in Machine Learning'
+zenodo_link: https://zenodo.org/record/...
+questions:
+- What is classification and how we can use classification techniques?
+objectives:
+- Learn classification background
+- Apply classification based machine learning algorithms
+- Learn ... (Simon)
```
Comment (anuprulez) [inline topics/statistics/tutorials/classification-machinelearning/tutorial.md]: Why is Simon's name present?

Comment (anuprulez) [reply topics/statistics/tutorials/classification-machinelearning/tutorial.md]: I am not sure why classification should be explained again while doing deep learning? They should be separate

File: topics/statistics/tutorials/deeplearning_intro/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (anuprulez) [inline topics/statistics/tutorials/deeplearning_intro/tutorial.md]: There is no text for deep learning I could find? I could find only images, data.yml files. I will create a fresh one.

**Comment by anuprulez** [comment]
@khanteymoori I recommend closing this PR. It is mixed with classification tutorial and I am not sure why it should be included in the tutorial of introduction to deep learning.
I will create a fresh one and will reuse some images from this draft.

**Comment by khanteymoori** [comment]
Sorry, It is a problem with tutorial.md file, and mixed with classification tutorial, so I closed.

**Pull request closed without merging**